pkg/io/grpc.go

```diff
@@ -232,6 +232,8 @@ wait:
 			return ctx.Err()
 		case <-master.Context().Done():
 			return master.Context().Err()
+		case <-s.exitCode:
```

Comment:
Should revert this, not needed. Stream end marks master context end, and that’s enough.